client/src/elm/Gizra/NominalDate.elm

```diff
@@ -95,8 +95,9 @@ fromLocalDateTime =
 
 {-| Given a date, return date representing it's last month day.
 
-    toLastDayOfMonth 2019-08-01" --> 2019-08-31"
-    toLastDayOfMonth 2017-02-20" --> 2017-02-28"
+    toLastDayOfMonth 2019 - 8 - 1 " --> 2019-08-31"
```

Comment:
Revert this

Reply:
removing the spacing is failing lint test

Reply:
@Diama1 

1. Revert to original state, and lets see if it still fails.
2. Error says file is not formatted properly with `elm-format` 0.8.7. So make sure you have `elm-format` 0.8.7 installed locally.

Reply:
@anvmn same error and when I run `elm-format ./src/elm --yes` it puts the spacing.

Reply:
@Diama1 

You have not reverted Nominal date change. We should not be seeing this in PR:

![Selection_327](https://github.com/TIP-Global-Health/eheza-app/assets/17882245/0410085e-36b8-4b22-a585-d7e008b23acb)

Reply:
done